### PR TITLE
Deployment scripts

### PR DESCRIPTION
**Pull request by josojo**
Same as other PR, but rebased on fork5

e2e tests:
- Do an actual deployment as it is said in the readme.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -49,14 +49,8 @@ jobs:
...
-      - uses: actions/cache@v2
-        with:
-          path: ${{ steps.yarn-cache.outputs.dir }}
-          key: ${{ matrix.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ matrix.os }}-yarn-
-      - run: yarn --frozen-lockfile
-      - run: yarn lint
\ No newline at end of file
+          node-version: 16
```
Comment (josojo) [inline .github/workflows/test.yml:52]: why again npm and not yarn. Document the reaons

File: deployment/1_deployforkableToken.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
+/* eslint-disable no-await-in-loop, no-use-before-define, no-lonely-if, import/no-dynamic-require, global-require */
```
Comment (josojo) [inline deployment/1_deployforkableToken.js:1]: Describe where the code is from

File: deployments/goerli_1692531859/deploy_output.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (josojo) [inline deployments/goerli_1692531859/deploy_output.json:1]: put that later into a separate PR

File: deployments/goerli_1692531859/deploy_parameters.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (josojo) [inline deployments/goerli_1692531859/deploy_parameters.json:1]: separate PR

File: development/contracts/dependencies/Getter.sol [hunk 5]
```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-3.0-only
```
Comment (josojo) [inline development/contracts/dependencies/Getter.sol:1]: more contract

**Review by edmundedgar — APPROVED** [review]

File: foundry.toml [hunk 6]
```diff
@@ -12,6 +12,7 @@ env = "${NODE_URL}"
 mainnet = "https://mainnet.infura.io/v3/${INFURA_KEY}"
 rinkeby = "https://rinkeby.infura.io/v3/${INFURA_KEY}"
 goerli = "https://goerli.infura.io/v3/${INFURA_KEY}"
+goerli = "https://sepolia.infura.io/v3/${INFURA_KEY}"
```
Comment (edmundedgar) [inline foundry.toml:15]: should be sepolia = 
?

**Pull request merged**